Resources/config/services.xml

```diff
@@ -199,6 +199,38 @@
             <argument type="service" id="fos_rest.view_handler"/>
         </service>
 
+        <!-- Form Controller -->
+        <service id="sulu_form.form_controller" class="Sulu\Bundle\FormBundle\Controller\FormController" public="true">
+            <argument type="service" id="fos_rest.view_handler.default"/>
+            <argument type="service" id="security.token_storage"/>
+            <argument type="service" id="sulu_form.manager.form"/>
+            <argument type="service" id="sulu_core.doctrine_rest_helper"/>
+            <argument type="service" id="sulu_core.doctrine_list_builder_factory"/>
+            <argument type="service" id="sulu_core.list_builder.field_descriptor_factory"/>
+            <argument type="service" id="sulu_core.list_rest_helper"/>
+        </service>
+
+        <!-- Form Website Controller -->
+        <service id="sulu_form.form_website_controller" class="Sulu\Bundle\FormBundle\Controller\FormWebsiteController" public="true">
+            <argument type="service" id="request_stack"/>
+            <argument type="service" id="form.registry"/>
+            <argument type="service" id="form.factory"/>
+            <argument type="service" id="sulu_form.configuration.form_configuration_factory"/>
+            <argument type="service" id="sulu_form.handler"/>
+            <argument type="service" id="security.csrf.token_manager"/>
+            <argument>%sulu_form.ajax_templates%</argument>
+            <argument>%sulu_form.static_forms%</argument>
```

Comment:
should be a service subscriber and only be registered in website context:

should be only register in admin context:

```xml
<tag name="sulu.context" context="website"/>
```